app/views/tracks.js

```diff
@@ -11,6 +11,109 @@ const TracksView = Mn.CompositeView.extend({
 
     childView: TrackView,
 
+    events: {
+         'mousedown .drag': 'dragStart'
+    },
+
+    initialize() {
+        this.dragObj = {
+            'dragDown': false,
+            'model': undefined,
+            'newIndex': undefined
+        };
+        this.listenTo(this.collection, 'sync', this.bindDragEvents);
+    },
+
+    bindDragEvents() {
```

Comment:
The tracksview is destroyed only when the tab is closed, so no need to unbind the listener i think.